src/libUtils/SWInfo.cpp

```diff
@@ -172,27 +174,91 @@ bool SWInfo::operator!=(const SWInfo& r) const { return !(*this == r); }
 /// Getters.
 const uint32_t& SWInfo::GetZilliqaMajorVersion() const {
   return m_zilliqaMajorVersion;
-};
+}
+
 const uint32_t& SWInfo::GetZilliqaMinorVersion() const {
   return m_zilliqaMinorVersion;
-};
+}
+
 const uint32_t& SWInfo::GetZilliqaFixVersion() const {
   return m_zilliqaFixVersion;
-};
+}
+
 const uint64_t& SWInfo::GetZilliqaUpgradeDS() const {
   return m_zilliqaUpgradeDS;
-};
-const uint32_t& SWInfo::GetZilliqaCommit() const { return m_zilliqaCommit; };
+}
+
+const uint32_t& SWInfo::GetZilliqaCommit() const { return m_zilliqaCommit; }
+
 const uint32_t& SWInfo::GetScillaMajorVersion() const {
   return m_scillaMajorVersion;
-};
+}
+
 const uint32_t& SWInfo::GetScillaMinorVersion() const {
   return m_scillaMinorVersion;
-};
+}
+
 const uint32_t& SWInfo::GetScillaFixVersion() const {
   return m_scillaFixVersion;
-};
-const uint64_t& SWInfo::GetScillaUpgradeDS() const {
-  return m_scillaUpgradeDS;
-};
-const uint32_t& SWInfo::GetScillaCommit() const { return m_scillaCommit; };
+}
+
+const uint64_t& SWInfo::GetScillaUpgradeDS() const { return m_scillaUpgradeDS; }
+
+const uint32_t& SWInfo::GetScillaCommit() const { return m_scillaCommit; }
+
+static size_t WriteString(void* contents, size_t size, size_t nmemb,
+                          void* userp) {
+  ((string*)userp)->append((char*)contents, size * nmemb);
+  return size * nmemb;
+}
+
+bool SWInfo::IsLatestVersion() {
+  string curlRes;
+  auto curl = curl_easy_init();
+  curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);
+  curl_easy_setopt(curl, CURLOPT_URL,
+                   "https://api.github.com/repos/Zilliqa/Zilliqa/tags");
```

Comment:
Suggest to give this URL a meaningful definition, in case we need to change it in the future.